salt/client/__init__.py

```diff
@@ -567,10 +567,14 @@ def cmd_batch(
         if 'gather_job_timeout' in kwargs:
             opts['gather_job_timeout'] = kwargs['gather_job_timeout']
 
+        eauth=kwargs.get('eauth', {})
```

Comment:
Oh, and since that method returns a generator you can view the results via:

```py
ret = local.cmd_batch('*', 'test.ping')
for i in ret: print i
```